index.html

```diff
@@ -1,33 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <link href="css/style.css" type="text/css" rel="stylesheet">
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Instagram</title>
 </head>
 <body>
-  Instagram
+<div class="backgroundColor">
+  <img id="brand" src="img/brand.png">
+  <img id="frame" src="img/frame.png">
+  <img id="home" src="img/home.png">
+  <img id="phones" src="img/phones.png">
 
-  Capture and Share the World's Moments
+  <h2 id="captNshare">Capture and Share</h2>
```

Comment:
I don't think there's a need to split that sentence. 

Check out Leon's work if you need a reference, his userid is fupuchu